src/AzureCacheRedisClientTests/RedisDbTests.cs

```diff
@@ -56,6 +56,38 @@ public async Task UsageTest()
             Assert.AreEqual(item, cachedItem);
         }
 
+        [TestMethod]
```

Comment:
Please add `[TestCategory("Integration")]`

Reply:
@willvelida, you may need to merge the Test project changes I have made on main for the tests to work (I have just been fixing them)

Reply:
TestCategory annotation has been applied to both `DeleteKey_SuccessfullyRemovesKey ` and `DeleteKey_IgnoresKeyWhenItDoesNotExist `